docker/hive/Dockerfile

```diff
@@ -15,17 +15,17 @@ RUN apt-get update && apt-get install --no-install-recommends -y patchelf && rm
 # Depending on the platform, the path to the binaries will be different
 # These values need to be updated if the version of ApiBara Starknet or Sink change.
 # platform: linux/amd64:
-#   - APIBARA_STARKNET_BIN_DIR: ns4qwsl6fgbv7mxhxpnaqhd66wnic8i6
+#   - APIBARA_STARKNET_BIN_DIR: 3l93dydg7m71r66x5wllf1j9czvj7zdk
 #   - APIBARA_SINK_BIN_DIR: 81f00xxzyxs6ih6376cw0qbznf6cl6bn
 # platform: linux/arm64
-#   - APIBARA_STARKNET_BIN_DIR: 3fz40zh8pscx7wp54nb78wm6p5y2v3g0
+#   - APIBARA_STARKNET_BIN_DIR: ksmcmb1ybrij98hjy4q54v31ag7d2l6l
 #   - APIBARA_SINK_BIN_DIR: ww2ghdaw1xpx42cf93zk5n3h222438ia
 ARG APIBARA_STARKNET_BIN_DIR
 ARG APIBARA_SINK_BIN_DIR
-# Run `docker image inspect apibara/starknet:1.5.0-x86_64` to get the exact path
-# Run `docker image inspect apibara/starknet:1.5.0-aarch64` to get the exact path
+# Run `docker image inspect apibara/starknet:1.6.0-x86_64` to get the exact path
+# Run `docker image inspect apibara/starknet:1.6.0-aarch64` to get the exact path
 # ⚠️ This path is subject to change, so it's important to check it before building the image ⚠️
-COPY --from=apibara /nix/store/${APIBARA_STARKNET_BIN_DIR}-apibara-starknet-1.5.0/bin/apibara-starknet /usr/local/bin/starknet
+COPY --from=apibara /nix/store/${APIBARA_STARKNET_BIN_DIR}-apibara-starknet-1.6.0/bin/apibara-starknet /usr/local/bin/starknet
```

Comment:
We should modify `starknet:1.5.0` to `starknet:1.6.0` in docker-compose.prod.yaml, docker-compose.yaml and docker-compose.staging.yaml no?

Reply:
hmmmm I would rather do this in another PR where we actually check that all the dockers are ok... right now we are no stop releasing and fixing things to quickly reach testnet, once this cools down we can go over all of the dockers.

Reply:
I don't even know if `1.6.0` works with `docker-compose.yaml` for example

Reply:
Ok let's do in subsequent PR

Reply:
https://github.com/kkrt-labs/kakarot-rpc/issues/1437